constants/route.ts

```diff
@@ -0,0 +1,8 @@
+export const ROUTE_PATHS = {
+  home: "/",
```

Comment:
이런식으로 표현해줘도 좋을 것 같아요~
```suggestion
  HOME: "/",
```